src/py42/modules/securitydata.py

```diff
@@ -52,6 +59,39 @@ def get_plan_security_events(
         min_timestamp=None,
         max_timestamp=None,
     ):
+        """Gets legacy security events.
+
+        Args:
+            plan_storage_info (:class:`py42.sdk.modules.securitydata.PlanStorageInfo`):
+                Information about storage nodes for a plan to get security events for.
+            cursor (str, optional): A cursor position for only getting events you did not
+                previously get. Defaults to None.
+            include_files (bool, optional): Whether to include the files related to the security
+                events. Defaults to None.
+            event_types: (str, optional): A comma-separated list of event types to filter by.
+
+                    Available options are:
+                        - 'DEVICE_APPEARED'
+                        - 'DEVICE_DISAPPEARED'
+                        - 'DEVICE_FILE_ACTIVITY'
+                        - 'PERSONAL_CLOUD_FILE_ACTIVITY'
+                        - 'RESTORE_JOB'
+                        - 'RESTORE_FILE'
+                        - 'FILE_OPENED'
+                        - 'RULE_MATCH'
+                        - 'DEVICE_SCAN_RESULT'
+                        - 'PERSONAL_CLOUD_SCAN_RESULT'
```

Comment:
I've been enclosing string literals in code markers to make them look a bit better in the sphinx docs: 
```
- ``DEVICE_APPEARED``
- ``DEVICE_DISAPPEARED``
- ``ETC``
```
We'll probably want to standardize. Calling @annie-payseur for an IX opinion.

Reply:
I am on board

Reply:
Yes, I think this is a good move. (code markers & standardizing)